src/shared/ml-thirdparty-client/index.ts

```diff
@@ -20,56 +20,47 @@
 
  * Google
  - Steven Wijaya <stevenwjy@google.com>
+ - Raman Mangla <ramanmangla@google.com>
  --------------
  ******/
 
 import { Simulator } from '~/shared/ml-thirdparty-simulator'
 import { PartyIdType } from './models/core'
 import { ThirdPartyTransactionRequest } from './models/openapi'
-
-namespace Client {
-  /**
-   * An interface definition for the configuration needed to setup the 
-   * Mojaloop client.
-   */
-  export interface Config {
-    /**
-     * Mojaloop URL for the client to communicate with.
-     */
-    mojaloopUrl: string
-  }
-}
-
-/**
- * Default configurations for the mojaloop client libary.
- */
-const defaultConfig: Client.Config = {
-  mojaloopUrl: '',
-}
+import { thirdpartyRequests, mojaloopRequests } from './requests'
+import {
+  ThirdpartyRequests,
+  MojaloopRequests,
+} from '@mojaloop/sdk-standard-components'
 
 /**
  * A client object that abstracts out operations that could be performed in
- * Mojaloop. With this, a service does not need to directly specify the request 
- * endpoint, body, params, and headers that are required to talk with the 
+ * Mojaloop. With this, a service does not need to directly specify the request
+ * endpoint, body, params, and headers that are required to talk with the
  * Mojaloop APIs. Instead, the service implementation could just pass the necessary
  * config upon initialization and relevant information in the function parameters
  * when it wants to perform a certain operation.
  */
 export class Client {
-  config: Client.Config
+  // config: Client.Config
   simulator?: Simulator
+  thirdparty: ThirdpartyRequests
+  mojaloop: MojaloopRequests
 
-  public constructor(config?: Client.Config) {
-    if (config) {
-      this.config = { ...defaultConfig, ...config }
-    } else {
-      this.config = defaultConfig
-    }
+  public constructor() {
+    // if (config) {
```

Comment:
why do you remove the config from the constructor? I think we still need to pass in the config from the constructor since we want to model this as a plugin, so it does not read from a config file directy

Reply:
yeah this constructor feels a little confusing